db/seeds.rb

```diff
@@ -17,28 +17,45 @@
 # These are not necessary to run the application, but required for full operation
 
 Group.create([
-  { id: 369, name: 'SVIE', type: 'bizottság' },
-  { id: 1, name: 'Kollégiumi bizottság', type: 'csoport' },
-  { id: 146, name: 'Reszortvezetők Tanácsa', type: 'csoport', parent_id: 369 },
-  { id: 16, name: 'Simonyi Károly Szakkollégium', type: 'reszort', parent_id: 146 },
-  { id: 106, name: 'KIR fejlesztők és üzemeltetők', type: 'szakmai kör', parent_id: 16,
+  { id: Group::SVIE_ID, name: 'SVIE', type: :team },
+  { id: Group::KB_ID, name: 'Kollégiumi bizottság', type: :group },
+  { id: Group::RVT_ID, name: 'Reszortvezetők Tanácsa', type: :group, parent_id: Group::SVIE_ID },
+  { id: Group::SIMONYI_ID, name: 'Simonyi Károly Szakkollégium', type: :team, parent_id: Group::RVT_ID },
+  { id: Group::KIRDEV_ID, name: 'KIR fejlesztők és üzemeltetők', type: :group, parent_id: Group::SIMONYI_ID,
     description: 'A Villanykari Információs Rendszer fejlesztésével és üzemeltetésével foglalkozó kör.',
     webpage: 'http://kir-dev.sch.bme.hu', maillist: 'kir-dev@sch.bme.hu', founded: 2001, issvie: true,
     delegate_count: 1, users_can_apply: true }
   ])
 
 User.create({ firstname: 'Júzer', lastname: 'Mezei', screen_name: 'mezei123' })
 
-leader_user = User.create({ firstname: 'Bálint Martin', lastname: 'Király', screen_name: 'kiraly96'})
-leader_membership = Membership.create({ group_id: 106, user_id: leader_user.id })
-Post.create({ membership_id: leader_membership.id, post_type_id: Membership::LEADER_POST_ID })
-Post.create({ membership_id: leader_membership.id, post_type_id: Membership::PEK_ADMIN_ID })
 
-rvt_member =  User.create({ firstname: 'Elnök', lastname: 'Simonyi', screen_name: 'elnokiugy' })
-Membership.create({ group_id: 146, user_id: rvt_member.id })
-
-kb_user = User.create({ firstname: 'Srác', lastname: 'KBs', screen_name: 'kbs123' })
-Membership.create({ group_id: 1, user_id: kb_user.id })
+# Create leaders for the groups
+[
+  { firstname: 'Elnok', lastname: 'Rvt', screen_name: 'rvt_elnok', group_id: Group::RVT_ID },
+  { firstname: 'Elnok', lastname: 'Svie', screen_name: 'svie_elnok', group_id: Group::SVIE_ID },
+  { firstname: 'B. Martin', lastname: 'Kircsi', screen_name: 'kiraly96', group_id: Group::KIRDEV_ID },
+  # { firstname: 'Bálint Martin', lastname: 'Király' , screen_name: 'kiraly96', group_id: Group::KIRDEV_ID },
```

Comment:
Why is this line commented out?

Reply:
I deleted both of lines and changed it to an anonymized user.